testproj/requirements.txt

```diff
@@ -0,0 +1,13 @@
+coreapi>=2.3.3
+coreschema>=0.0.4
+Django==2.0
+django-cors-headers>=2.1.0
+django-filter==1.1.0
+djangorestframework==3.7.7
+djangorestframework-camel-case>=0.2.0
+future>=0.16.0
+inflection>=0.3.1
+openapi_codec>=1.3.2
+Pillow==4.3.0
+Pygments==2.2.0
+ruamel.yaml>=0.15.34
```

Comment:
Half of the requirements here are for drf_yasg itself, and the other half are listed in /requirements/test.txt...

Maybe it would make sense to move the testproj requirements into a requirements,txt file in testproj and reference that from test.txt, but duplicating them is not acceptable.